slides/03-what-makes-a-model.qmd

```diff
@@ -51,7 +51,7 @@ countdown(minutes = 3)
 ::: columns
 ::: {.column width="40%"}
 
--   Choose a model
+-   Choose a [model]{.underline}
```

Comment:
Underlining seems to show up a lot better to highlight specific parts of a slide